### ISSUE TITLE
AboutModal:  aria-label on the close button can not be overridden

### ISSUE THREAD

**Issue by tlabaj** [patternfly/patternfly-react#3855]
**Describe the issue. What is the expected and unexpected behavior?**
The aria-label in the About Modal is hard coded.  The consumer should be able to override it.
